static/scss/base/vertical-nav.scss

```diff
@@ -0,0 +1,30 @@
+$predecessor: rgb(137, 179, 157);
+$active: rgb(230, 178, 87);
+
+nav[aria-labelledby="knowledge-domain-navigation"] {
```

Comment:
Hmmmm....these styles are quite rich, but can be distracting from the main content. Let's review together on Monday.

Reply:
Yeah, agreed. Thanks for looking, and lets review Mon.